vendor/src/github.com/docker/engine-api/types/container/host_config.go

```diff
@@ -271,6 +272,22 @@ type UpdateConfig struct {
 	RestartPolicy RestartPolicy
 }
 
+// MountConfig holds attributes for configuring a container mount
+// This replaces the `Binds` field (kept for back-compat)
+// Not all fields are used in all cases, and some fields conflict based on `Type`
+type MountConfig struct {
+	Type        string // Type of mount, e.g. 'bind', 'volume'
+	Destination string // Path in the container to mount to
+	Source      string `json:",omitempty"` // Path on the host to mount from, only used for binds
+	Propagation string `json:",omitempty"` // [r]shared, r[private], only used for binds
+	Populate    bool   `json:",omitempty"` // Populate volume from container image path if the volume is empty
+	Writable    bool
+	// Sets selinux sharing context (binds only)
+	// Allows binds to either be allowed to be shared with other containers or private to the container
+	MCSAccessMode string                      `json:",omitempty"`
```

Comment:
I'm inclined to not include this... this is for people using the `z` and `Z` modes on binds (e.g. `-v /foo:/bar:Z`)
Since with this API we specifically don't create binds, it seems a little strange to allow modifying the security labels.